### DISCUSSION TITLE
it/italia-portale-govuk/

### DISCUSSION THREAD

**Discussion by giscus** [matteocontrini/blog#4] [Comments]
# it/italia-portale-govuk/

Il portale GOV.UK è il punto di accesso per tutte le informazioni e i servizi pubblici del Paese. In Italia non abbiamo nulla di vagamente simile, purtroppo.

https://blog.contrini.it/it/italia-portale-govuk/

**Comment by mauro334** [comment]
Tempo fa, non ricordo dove, parlavano di come in GB il fulcro di tutti i rapporti tra PA e non PA fosse l'informazione. All'epoca l'esempio erano le brochure messe un po' ovunque ad informare, oggi è gov.uk....

Il problema in Italia non è solo la ricerca delle informazioni, quasi impossibile per una persona comune, ma sarebbe il mantenere aggiornate queste informazioni dato che non esiste qualcuno il cui lavoro sia questo.

Non esiste né a livello statale né locale. Il nostro paese parla semplicemente una lingua diversa da quella della semplice logica comune. Hai bisogno di qualcosa? Sai che non la troverai nella PA.

Chiudo con un esempio, un annetto fa avevo bisogno di pagare un verbale dei Carabinieri... ebbene si poteva pagare solo ed esclusivamente presso un ufficio postale oppure in una sezione dedicata del sito delle Poste: roba da pazzi!

**Reply by matteocontrini** [reply]
Sui pagamenti fortunatamente ci sono stati dei progressi con PagoPA. C'è la questione dibattuta delle commissioni, ma non è che pagare i bollettini alle Poste sia gratis.

>ma sarebbe il mantenere aggiornate queste informazioni dato che non esiste qualcuno il cui lavoro sia questo.

Posso capire che sia un grande sforzo, ma al momento siamo allo zero assoluto. Faccio un esempio: nel momento in cui si decide di aumentare il limite di fatturato per il regime forfettario delle partita IVA (l'anno scorso è passato da 65mila a 85mila), è mai possibile non ci sia nessuno che si prenda la briga di scrivere questa informazione da qualche parte? Non esiste **letteralmente** una pagina di nessun sito del governo dove questa informazione sia indicata.

Persino la Grecia, che in quanto a digitalizzazione è messa molto peggio di noi, fa meglio, con un portale in due lingue (https://www.gov.gr/en/).

**Reply by mauro334** [reply]
Mi sono un attimo fatto coraggio ed ho cercato "pagamento verbale Carabinieri". Ti invito a fare un attimo questa ricerca e deliziarti anche tu. 

Partendo dai certificati del sito carabinieri.it e finendo con il leggere la procedura per pagare online.

**Comment by blackne0n** [comment]
Il budget del GDS (che gestisce anche il servizio gov.uk) riceve un budget enorme (ma usato molto bene, visto che la maggior parte dei siti del governo rispettano i guidelines sviluppati dal GDS. potrebbe essere una controparte dell'AgID?

Nel UK apri una società in un'ora, in Italia ci metti una settimana come minimo. Nel UK rinnovi il passaporto online, fai le procedure per la patente online, richiedi la maggior parte dei benefici da un singolo portale, puoi addirittura segnalare qualcosa di non urgente alla polizia: tutto dallo stesso portale.


https://www.itpro.com/government-it-strategy/25656/government-digital-service-gets-450m-mega-budget

**Reply by matteocontrini** [reply]
>potrebbe essere una controparte dell'AgID?

Non direi, l'AgID è una struttura grossa ma alla fine produce solo PDF e linee guida, di operativo fa poco e nulla. A memoria in passato c'erano stati anche battibecchi perché Renzi voleva abolirla e sostituirla con il Team/Dipartimento per la Trasformazione Digitale, che è invece l'equivalente del GDS ed è un ottimo esempio del fatto che le cose si possono fare bene.

È nato nel 2016 e lavora come una startup puntando molto sull'open source. Da lì sono usciti l'app IO, i kit di design e tanti altri progetti. Hanno spinto anche per realizzare un portale di documentazione pubblico (docs.italia.it), che AgID puntualmente non usa perché preferisce i PDF violando probabilmente le sue stesse linee guida.

Purtroppo ho l'impressione che il team sia stato poi un po' bistrattato e ad ogni governo è cambiato il coordinamento: prima c'era un ministro, ora non c'è più un ministro. Il sito del team non esiste più perché si è fuso con quello del ministro/sottosegretario, che sembra però più una passerella del sottosegretario di turno.

Con tutti i limiti di un'osservazione esterna a me sembra molto evidente bisognerebbe puntare molto di più sul dip. trasformazione digitale. Ogni cosa buona che riguardi il digitale nel pubblico dell'ultimo decennio è uscita da lì. E intanto invece si creano nuove software house ([3-I SpA](https://www.wired.it/article/3-i-spa-inps-inail-istat-ricerca-personale-piano-industriale/)) che dopo anni non hanno ancora nemmeno deciso che cosa devono fare.

Al pasticcio si aggiunge anche il fatto che [ogni ente pubblico](https://matteosonoio.it/it/analisi-pa-2024/) ha la sua struttura tecnica ed escono a volte di quelle porcherie...
